widgets/modules/EnsEMBL/Web/Apache/Static.pm

```diff
@@ -0,0 +1,36 @@
+package EnsEMBL::Web::Apache::Static;
+
+use strict;
+use warnings;
+
+sub add_caching_headers {
```

Comment:
decided to handle this using $self->PREV::add_caching_headers in code review.

Reply:
Discovered that we can't. I extracted a method from `add_caching_headers` specifically to check whether the response to a particular url needs to be cached, and am overriding it in public plugins. The logic that adds caching headers will remain unmodified in `ensembl-webcode`.